Added context support (#39)
AppsFlyer/go-sundheit: README.md

	},
	gosundheit.InitialDelay(0),
	gosundheit.ExecutionPeriod(2 * time.Minute),

>     gosundheit.ExecutionTimeout(5 * time.Second)

Reviewer: nit: indentation

>     gosundheit.ExecutionTimeout(5 * time.Second)

Reviewer: ```suggestion
	gosundheit.ExecutionTimeout(5 * time.Second),
```

}
```

> And register our custom check, scheduling it to run after 1 sec, every 30 sec, with a 5 seconds timeout:

Reviewer: ```suggestion
And register our custom check, scheduling it to run every 30 seconds (after a 1 second initial delay) with a 5 seconds timeout:
```

	Lottery{myname: "custom.lottery.check", probability:0.3},
	gosundheit.InitialDelay(1*time.Second),
	gosundheit.ExecutionPeriod(30*time.Second),

> 	gosundheit.ExecutionTimeout(5*time.Second)

Reviewer: ```suggestion
	gosundheit.ExecutionTimeout(5*time.Second),
```
